CHANGELOG

```diff
@@ -1,3 +1,7 @@
+### 0.3.6 - FEB.06.2017
```

Comment:
This is still wrong, but I can fix it before I release it.  Needs to be a minor version bump, not a patch level version bump.  MAJOR.MINOR.PATCH ;)